### PR TITLE
[#7915] fix - Text for empty chat state is shown upside down... by upgrading RN to 0.59.3

### LINKED ISSUE #7915: Text for empty chat state is shown upside down and text direction is strange
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: After joining public chat or 1:1 chat its text and icon are shown upside down. Only after sending there at least 1 message the text and icon are shown normally
Screenshot_20190404-154848

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
text and icon have usual position

#### Actual behavior
[comment]: # (Describe what actually happened.)
text and icon are upside down
### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

- Open Status
- Tap +
- Tap Join public chat
- for topic: type "public" (or any other name of empty chat) and tap enter
- check text and icon in the open empty chat

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: 0.12.0
* Operating System: Android and iOS

### PR DESCRIPTION
**Pull request by bitsikka**
fixes #7915 

### Summary:

Fixes by upgrading RN from 0.59.2 to 0.59.3

status: ready

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]

File: mobile_files/package.json.orig [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "querystring-es3": "0.2.1",
     "react": "16.8.3",
     "react-dom": "16.4.2",
-    "react-native": "git+https://github.com/status-im/react-native.git#status-v0.59.2",
+    "react-native": "0.59.3",
```
Comment (rasom) [inline mobile_files/package.json.orig]: https://github.com/facebook/react-native/issues/24001 is not included into 0.59.3, we still have to use our fork

**Review by rasom — CHANGES_REQUESTED** [review]

File: mobile_files/package.json.orig (hunk 1 above)
Comment (bitsikka) [reply mobile_files/package.json.orig]: Thanks for spotting that! It's a pretty significant regression.

So the path forward is to upgrade our fork to 0.59.3 with the commit that fixes the long press bug? 🤔 

Should I submit PR to status-im/react-native and update this PR accordingly?

Comment (rasom) [reply mobile_files/package.json.orig]: I've just updated current PR

**Review by rasom — APPROVED** [review]

File: mobile_files/package.json.orig (hunk 1 above)
Comment (bitsikka) [reply mobile_files/package.json.orig]: thank you!

**Comment by churik** [comment]
@bitsikka thanks for your fix!

There are several issues.

#### 1) No label is shown if you open chat via valid deep link

**Steps:**
- tap on valid deep link (i.e. https://get.status.im/chat/public/coo)
- pay attention to opened chat

**Expected result:** can see icon \ label for empty screen state
**Actual result:** 
photo_2019-04-08 12 07 10
If you tap back and again open chat, you can see the label. On nightly you can see label right after you open chat via deep link.

#### 2) Username (chat name) is not centered when it contains > 1 row
I expect that username should be aligned on center.
For now:
dsfdsf

**Comment by rasom** [comment]
@churik can't `1)` be reproduced in `develop`?

`2)` is certainly presented in `develop`, so I would fix this in a separate issue

**Comment by churik** [comment]
@rasom 
1) is not in `develop`
2) yes, it is in `develop`. Will create separate issue for it.

**Comment by rasom** [comment]
`1)` can be reproduced in nightly if another chat was opened before opening the link

**Comment by rasom** [comment]
@churik it seems that both `1)` and `2)` are not introduced in this PR

**Comment by churik** [comment]
@rasom yes, agree, steps a bit different.
Will create separate issue for them.

**Comment by bitsikka** [comment]
On it, `1)` might have to do with a bug in #7454 I'm submitting the PR for `2)` is something different (should be a simple fix)

Investigate both right away

**Comment by churik** [comment]
Thank you @bitsikka !
I'll create separate issues for them.

**Comment by rasom** [comment]
@bitsikka please consider filing separate PRs for these issues, I would prefer to merge this one asap :)

**Comment by bitsikka** [comment]
> @bitsikka please consider filing separate PRs for these issue, I would prefer to merge this one asap :)

definitely

separate PRs for `1)` and `2)` comping up

**Comment by churik** [comment]
@bitsikka 
Issues:
- https://github.com/status-im/status-react/issues/7930
- https://github.com/status-im/status-react/issues/7931

Tested:
- empty state for public chats opened: via deep link, via link inside app, via "Join public chat"
- empty state for group chat: when it is created as result of sync, via "Create group chat", as a member and as admin
- empty state for 1-1 chat: when scanning QR, pasting valid key, starting chat with ENS username,  via deep link, via user's profile -> Send message

**Pull request merged**